packages/pv-scripts/scripts/prod.js

```diff
@@ -46,6 +49,21 @@ function webpackBuild(webpackConfig) {
         return reject(new Error(messages.errors.join("\n\n")));
       }
 
+      // generate the css stats
+      if (process.env.PV_WEBPACK_STATS) {
+        const destPath = getAppConfig().destPath;
+        explore(`${path.resolve(".", destPath, "css")}/*.css`, {
```

Comment:
maybe path.resolve(process.cwd(), destPath, "css") would be more stable

Reply:
it resolves from the current working directory but you are right, it reads better when it says `cwd` instead of "."